baseband/tests/coveragerc

```diff
@@ -28,4 +28,7 @@ exclude_lines =
    def main\(.*\):
 
    # Ignore branches that don't pertain to this version of Python
-   pragma: py{ignore_python_version}
\ No newline at end of file
+   pragma: py{ignore_python_version}
+
+   # Don't complain about IPython completion helper
+   def _ipython_key_completions_
```

Comment:
missing eol

Reply:
It actually didn't have one before.  Not sure if it's auto-eliminated; I tried adding one, and it gets deleted.

Reply:
It's there now!